testing/zerotier-one/zerotier-one.confd

```diff
@@ -0,0 +1 @@
+zerotier_opts=""
```

Comment:
I don't believe so, it's useful for documentation. Most files we have in `/etc/conf.d` do nothing functional, but document the possible variables. Even if we were to go with `command_args`, we can't expect end-users to know about OpenRC internals. :)